apps/aehttp/src/aehttp_app.erl

```diff
@@ -121,14 +121,19 @@ get_and_parse_ip_address_from_config_or_env(CfgKey, App, EnvKey, Default) ->
     {ok, IpAddress} = inet:parse_address(binary_to_list(Config)),
     IpAddress.
 
-get_http_api_acceptors(_) -> ?INT_ACCEPTORS_POOLSIZE.
+get_http_api_acceptors(external) ->
+    aeu_env:user_config_or_env([<<"http">>, <<"external">>, <<"acceptors">>],
+                               aehttp, [external, acceptors], ?INT_ACCEPTORS_POOLSIZE);
```

Comment:
I guess using the same constant for internal/external pool size is intended.

Reply:
Yes, that's correct.